### DISCUSSION TITLE
How to install VISIDATA under windows 10 without admin privileges?

### DISCUSSION THREAD

**Discussion by duemme** [saulpw/visidata#901] [Q&A]
Hello,
I would like to use VISIDATA on my enterprise laptop that runs WIN 10 but I don't have admin privileges on it.
Is it possible to install VISIDATA (and all its dependencies) as a normal user?
If yes, can someone help me achive this?
Thank you

**Comment by aborruso** [comment]
Ciao @duemme Windows is not yet directly supported.

It is much better to use it inside a Linux OS. If you have Win, you could [install](https://docs.microsoft.com/en-us/windows/wsl/install-win10) Windows Subsystem for Linux (but I do not know if you can install it).

**Reply by duemme** [reply]
Seems that admin privileges are required to install WSL. Thank you @aborruso
